### PR TITLE
specify sidecar resource requirements

### PR DESCRIPTION
**Pull request by chlgit**
How sophisticated we need here? I am not sure. We have a plan to retire hacheck sidecar, and it's the only sidecar we use now.

### CODE REVIEW

File: paasta_tools/kubernetes_tools.py [hunk 1]
```diff
@@ -678,6 +679,12 @@ def get_resource_requirements(self) -> V1ResourceRequirements:
             },
         )
 
+    def get_sidecar_resource_requirements(self) -> V1ResourceRequirements:
+        return V1ResourceRequirements(
+            limits={"cpu": 0.1, "memory": "1024Mi", "ephemeral-storage": "1024Mi"},
```
Comment (vkhromov) [inline paasta_tools/kubernetes_tools.py]: `ephemeral-storage` better should be set to a real value, as k8s will reserve these `1024Mi` at a node and will not schedule anything there if there will be not enough `ephemeral-storage` there.
E.g. if the sidecar doesn't use the local disk for temporary files then `ephemeral-storage` should be `0Mi`.

Comment (chlgit) [reply paasta_tools/kubernetes_tools.py]: Ah, I see.

Comment (qui) [reply paasta_tools/kubernetes_tools.py]: I think hacheck needs a little bit of local disk, for writing files to /var/spool during draining, but probably a very small amount.

Comment (chlgit) [reply paasta_tools/kubernetes_tools.py]: Let's start from 256Mi for the disk and revisit it if needed.

**Pull request merged**